source/sync_serial.cpp

```diff
@@ -31,17 +39,33 @@ static PVoidCallback irq_callback[NUM_SYNC_SERIAL];
 static void *irq_callback_data[NUM_SYNC_SERIAL];
 static uint8_t used_serials;
 
+// To be able to select a specific device
+void *allocate_sync_serial(void* device)
```

Comment:
maybe we should rename this to allocate_serial() ? UARTE is async not sync...

Reply:
Oh, you are right.
I think it would be better to create a separate file (like async_serial.)
I'll fix this.

Reply:
you can have one file for both, you can rename them all; could be alloc_peripheral() or alloc_serial() as you wish

Reply:
Done in commit c030199.